MyJobApplication/src/com/amaderorg/myjobapp/Model/Database/Tables/ContactInformation.java

```diff
@@ -0,0 +1,74 @@
+package com.amaderorg.myjobapp.Model.Database.Tables;
+
+/**
+ * Created by souvi_000 on 3/28/2016.
+ */
+public class ContactInformation {
```

Comment:
This class can be renamed as 'Contact'. Also please add documentation for this class, member variables and methods.